pkg/factsengine/entities/facts_gathered.go

```diff
@@ -36,6 +40,16 @@ func (e *FactGatheringError) Wrap(msg string) *FactGatheringError {
 	}
 }
 
+func (e *Fact) Prettify() (string, error) {
+	prettifiedValue, err := Prettify(e.Value)
+	if err != nil {
+		return "", errors.Wrap(err, "Error pretty-printing fact value data")
```

Comment:
Maybe better the error being `Error prettifying fact value data`

Reply:
Done :+1: